cmd/sks_nodepool_add.go

```diff
@@ -145,8 +145,8 @@ func init() {
 	sksNodepoolAddCmd.Flags().Int64("disk-size", 50,
 		"Nodepool Compute instances disk size")
 	sksNodepoolAddCmd.Flags().StringSlice("anti-affinity-group", nil,
-		"Nodepool Anti-Affinity Group <name | id> (can be specified multiple times)")
+		"Nodepool Anti-Affinity Group NAME|ID|NAME|ID (can be specified multiple times)")
 	sksNodepoolAddCmd.Flags().StringSlice("security-group", nil,
-		"Nodepool Security Group <name | id> (can be specified multiple times)")
+		"Nodepool Security Group NAME|ID|NAME|ID (can be specified multiple times)")
```

Comment:
Erroneous repetition?
```suggestion
		"Nodepool Security Group NAME|ID (can be specified multiple times)")
```